ui/css/treeherder-job-buttons.css

```diff
@@ -3,7 +3,7 @@
  */
 
 .job-group {
-  margin: 0 -3px 0 3px;
+  margin: 0 0;
```

Comment:
I think you could shorthand this to a single `0` is that is the intent. Or remove the margin property entirely if the job group element would inherit a margin of 0 from elsewhere.